tutorials/observing_examples.ipynb

```diff
@@ -0,0 +1,372 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from synphot import Observation\n",
+    "import numpy as np\n",
+    "\n",
+    "obs = Observation(photlam_hatp11, agile)\n",
+    "obs.plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To get the expected count rate for a specific collecting area, use the <a href=\"https://synphot.readthedocs.io/en/latest/api/synphot.observation.Observation.html#synphot.observation.Observation.countrate\">countrate()</a> method: "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "11852206.200075638 ct / s\n"
+     ]
+    }
+   ],
+   "source": [
+    "r = 3.5 / 2 * u.m  # radius of 3.5m ARC telescope at APO\n",
+    "area = np.pi * r.to(u.cm) ** 2\n",
```

Comment:
Technically, `r.to(...)` is unnecessary. As you are passing in a Quantity, there is unit attached already and `synphot` should know how to handle it under the hood. And if you find out that it cannot, then it is a bug.

```python
area = np.pi * r ** 2
```

Reply:
Good to know! It does handle it.